extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -253,7 +253,10 @@ export class LiveListManager {
     );
 
     const inClientDomLiveList = this.liveLists_[id];
-    inClientDomLiveList.toggle(!liveList.hasAttribute('disabled'));
+    inClientDomLiveList.toggle(
+      !liveList.hasAttribute('disabled') &&
+        !liveList.hasAttribute('live-story-disabled')
```

Comment:
can you explain why we need a specific one for live story? why wouldnt `disabled` just work for that scenario?

Reply:
Done.